www/bank-account.html.spt

```diff
@@ -32,7 +32,7 @@ title = _("Bank Account")
 
 {% block heading %}
 
-    <h1>{{ status }}</h1>
+    <h1>{{ status|safe }}</h1>
```

Comment:
If this PR ensures that all i18n messages are properly escaped then we won't need to worry anymore about a translator purposefully or inadvertently causing an HTML injection.